assets/src/edit-story/app/story/save/saveElement.js

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
I wonder if `saveElement` would be better placed under `elements` folder? Currently, half of the element output is separated. Perhaps it's fine.

Also, I'm wondering if `Save` should be changed to `Output` instead. Two reasons:
- In another PR (#4167) the save and preview are sharing quite a lot -- like all the wrapper params for example, only the inner element (`p` or `amp-img` vs `img`) is different. `Output` would allow using `saveElement` for both, just create a separate `preview.js` for the inner element.
- `Save` isn't actually doing/allowing the `Save` action, `Output` might be more accurate to describe the component anyway.

Not a crucial thing here though, could do that in #4167, too. Thoughts?

Reply:
Yeah, I like `Output` better too. And, yeah, we could move `saveElement` as well. I do think they'll be the same for the most part. Not sure about preview right now, but the most drastic issue I discovered with `saveElement` and `renderToString` is that I wasn't able to use hooks inside of `Save`/`Output` templates. That partially makes sense and hopefully won't be an issue for `Preview` templates either. But basically, these render functions are not quite like other render functions (e.g. `Display` or `Edit`). Have you run into this as well?

Reply:
Moved this to https://github.com/ampproject/amp-wp/pull/4173

Reply:
> Have you run into this as well?

Nope, haven't noticed this issue so far, which hooks did you need to use within saving the output?

Note: we can continue the discussion in #4173

Reply:
Yep. I'll bring this back up in #4173